test/integration/lib/webpush_server.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const express = require('express');
+const Config = require('config-js');
+const getRawBody = require('raw-body');
+const assert = require('assert');
+const crypto = require('crypto');
+const ece = require('http_ece');
+const typer = require('media-typer');
+var config = new Config('./test/integration/lib/config/foxbox.js');
+
+var webpush_server = (function() {
+  var serverECDH = crypto.createECDH('prime256v1');
+  var serverPublicKey = serverECDH.generateKeys('base64');
+  var _server = express();
+  
+  var port = config.get('webpush.port');
+  var endpoint = config.get('webpush.endpoint');
+  
+  var incomingPushMsg;
+  var serverInstance;
+
+  function getEndpointURI() {
```

Comment:
Oh, that's a drawback of anonymous functions, then. I also see the function reexport that is a type of duplication (each time someones requires to add a function, he/she has to it at 2 different places).

Let's keep it as is for now, but that sounds like a good refactor to do. What do you think?